src/external/nginx.cr

```diff
@@ -30,6 +30,10 @@ module Wsman
         Dir.mkdir_p(result)
         result
       end
+
+      def cleanup_site(site_name)
```

Comment:
Since all this function does is removes the site configuration, please follow the naming convention of other methods an rename it to something like `delete_site_config`.